frontend/src/components/page-about/index/leadership.module.css

```diff
@@ -0,0 +1,41 @@
+@config "../../../config/tailwind.config.js";
```

Comment:
I put this css file directly next to the component, since imo there's no point in putting it in `css/` if the styles here aren't ever going to be shared by other components. If they are going to be shared, then the styles should live in a file that is named accordingly (e.g. common.module.js) and live in `css/`